charts/airflow/values.yaml

```diff
@@ -328,6 +328,13 @@ airflow:
     ##
     extraVolumes: []
 
+    ## Enable shareProcessNamespace
+    ## https://kubernetes.io/docs/tasks/configure-pod-container/share-process-namespace/
+    shareProcessNamespace: false
+
+    ## Add extra containers
+    extraContainers: {}
```

Comment:
Please format these lines like the other docstrings.

Reply:
Also, we already have a PR that adds this feature: #456

Reply:
As https://github.com/airflow-helm/charts/pull/456 will add this feature, please remove this line.